src/ui.navigation-progress/index.vue

```diff
@@ -93,7 +126,21 @@ const realMax = computed(() => {
   return Number(props.max);
 });
 
-const progressPercent = computed(() => `${(props.value / realMax.value) * 100}%`);
+const isVariant = computed(() => ({
+  stepper: props.variant === VARIANT.stepper,
+  progress: props.variant === VARIANT.progress,
+}));
+
+const progressPercent = computed(() => {
+  const maxPercent = 100;
+  const currentPercent = (props.value / realMax.value) * maxPercent;
+
+  if (isVariant.value.progress) {
+    return `${currentPercent}%`;
+  }
+
+  return `${currentPercent} ${maxPercent}`;
```

Comment:
this is not clear for me, need to discuss

Reply:
It's applied to `<circle>` stroke-dasharray attribute when variant is Stepper, so we need this data format

Reply:
its better to divide this computed into 2 computed. and calculate `${currentPercent} ${maxPercent}` inside the component where it need to be used.